flow360/component/flow360_params/flow360_output.py

```diff
@@ -383,6 +394,10 @@ def schema_extra(schema, model):
                 schema["properties"]["outputFields"]["items"]["enum"], CommonFieldNamesFull
             )
 
+    @classmethod
+    def _get_widgets(cls) -> dict[str, str]:
+        return {"monitors/additionalProperties/monitorLocations/items": "vector3"}
+
```

Comment:
can we write a comment here that this widget is for ui schema for webUI

Reply:
Added comment, renamed schema-specific functions